src/components/KeyboardShortcutsModal.js

```diff
@@ -40,19 +40,50 @@ class KeyboardShortcutsModal extends React.Component {
             KeyboardShortcutsActions.showKeyboardShortcutModal();
         }, openShortcutModalConfig.descriptionKey, openShortcutModalConfig.modifiers, true);
 
-        const closeShortcutModalConfig = CONST.KEYBOARD_SHORTCUTS.ESCAPE;
-        this.unsubscribeEscapeModal = KeyboardShortcut.subscribe(closeShortcutModalConfig.shortcutKey, () => {
+        /**
```

Comment:
Just small detail, I believe this comments block should use `//` instead `/**` in the block comment.

Reply:
What's your idea behind using inline vs multiline codeblock?

Reply:
We mainly use multiline code blocks for the JSDoc and the prop-types comment. I couldn't find documentation, but I recall we've reviewed this before.

@robertjchen Could you help confirm this? 

I'm okay if we are not strict about it now, just something came up in my mind.

Reply:
Got you, thanks. Updated.